Procfile

```diff
@@ -1,2 +1,2 @@
-web: node server.js
-dev: nodemon server.js
+web: node serverDist/index.js
```

Comment:
Yeah because it's using Babel it's more performant to compile the ES6 code into ES5 files for production, instead of compiling it on the fly (which is what's happening while it runs in `NODE_ENV=development`). I called it `serverDist/` since there was already a `dist` directory for the client-side app.